expr.c

```diff
@@ -673,6 +678,22 @@ expr_eval_match(struct expr *ex, struct expr_eval_arg *ea)
 	return expr_eval_and(ex, ea);
 }
 
+static int
+expr_eval_mdflags(struct expr *ex, struct expr_eval_arg *ea)
+{
+	struct match *mh;
+	const char *flags;
+
+	mh = match_alloc(ex, ea->ea_msg);
+	flags = TAILQ_FIRST(ex->ex_strings)->val;
+	for (; *flags != '\0'; flags++)
+		message_flags_set(&ea->ea_msg->me_mflags, *flags, 1);
```

Comment:
Check for errors.

Reply:
Should it be err() in risk of loosing email?
I push -f'ed a warn() in the meantime.

Reply:
You should return EXPR_ERROR instead since that will cause evaluation to be aborted and the error to trickle all the way up.